docs/articles/documentation/guides/continuous-integration/circleci-and-lambdatest.md

```diff
@@ -0,0 +1,163 @@
+
+Open the **Environment Variables** category and click **Add Environment Variable**:
+
+![Click the Add Environment Variable button in Project Settings](../../../images/circleci-lambdatest/add-env-variable.png)
+
+In a separate browser tab or window, open the LambdaTest app and click **Automation** in the navigation bar:
+
+![Navigate to the Automation page in the LambdaTest app](../../../images/circleci-lambdatest/lambdatest-automation.png)
+
+Click ![Key icon](../../../images/circleci-lambdatest/lambdatest-key-icon.png) to display the access key info. Copy the *username*:
+
+![Copy the username from LambdaTest Access Key Info](../../../images/circleci-lambdatest/lambdatest-credentials.png)
+
+In CircleCI, enter `LT_USERNAME` as the variable name, paste the copied value below, and click **Add Environment Variable**:
+
+![Add the LT_USERNAME environment variable in CircleCI](../../../images/circleci-lambdatest/specify-secret-variable.png)
+
+In the same manner, copy the *access key* from LambdaTest and create the `LT_ACCESS_KEY` environment variable in CircleCI to store this key.
+
+## Step 5 - Install the LambdaTest Browser Provider and JUnit XML Reporter
+
+Install the following npm packages locally in your project directory:
+
+* [testcafe-browser-provider-lambdatest](https://www.npmjs.com/package/testcafe-browser-provider-lambdatest) – the LambdaTest browser provider that enables TestCafe to run tests in the LambdaTest cloud,
+* [testcafe-reporter-xunit](https://www.npmjs.com/package/testcafe-reporter-xunit) – the JUnit XML reporter that generates test run reports in the XML format. CircleCI can process XML reports and display test results in the UI.
+
+Run the following command to install both packages:
+
+```sh
+npm install --save-dev testcafe-browser-provider-lambdatest testcafe-reporter-xunit
+```
+
+Commit the `package-lock.json` file generated by npm into the repository. This file allows CircleCI to use a fast `npm ci` command to install the dependencies.
+
+## Step 6 - Add the `test` script to package.json
+
+To specify how CircleCI should launch TestCafe, add the `test` script to the project's `package.json` file. Use the `testcafe` command in the script to run tests on LambdaTest.
+
+```json
+"scripts": {
+  "test": "testcafe \"lambdatest:Chrome@85.0:Windows 10\" tests/**/* -r xunit:/tmp/test-results/res.xml"
+}
+```
+
+After you install the dependencies and specify the `test` script, `package.json` should look as follows:
+
+```json
+{
+  "scripts": {
+    "test": "testcafe \"lambdatest:Chrome@85.0:Windows 10\" tests/**/* -r xunit:/tmp/test-results/res.xml"
+  },
+  "devDependencies": {
+    "testcafe": "*",
+    "testcafe-reporter-xunit": "*",
+    "testcafe-browser-provider-lambdatest": "*"
+  }
+}
+```
+
+For more information on how to configure a test run with the `testcafe` command, see [Command Line Interface](../../reference/command-line-interface.md).
```

Comment:
After I push `package.json`, tests don't run for me. I deleted the project and added it back. Now I get the following error when I paste the config:

![image](https://user-images.githubusercontent.com/49588154/99641514-6fbd4e80-2a5b-11eb-9cc7-076feaf4bbb3.png)

Maybe we should go with manually adding `.circleci/config.yml`?

Reply:
UPD. I commited and pushed the `.circleci/config.yml`. When I try to build, I get the following error: 
![image](https://user-images.githubusercontent.com/49588154/99642020-1275cd00-2a5c-11eb-98e5-b0d50e0f15f2.png)

Reply:
Fixed by mentioning the right branch to push to...